### ISSUE TITLE
Major issue: not compatible with Rails >= 4.2 anymore

### ISSUE THREAD

**Issue by fabn** [Locale/localeapp#180]
Released 1.0.0 version is not currently installable with rails >= 4.2.0 because activesupport [requires i18n ~> 0.7](https://rubygems.org/gems/activesupport/versions/4.2.0) 

This Gemfile

```ruby
source 'https://rubygems.org'
ruby '2.1.5'
gem 'rails', '~> 4.2.0'
gem 'localeapp', '~> 1.0.0'
```

produces this output from `bundle install`

```
Fetching gem metadata from https://rubygems.org/...........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies.........
Bundler could not find compatible versions for gem "i18n":
  In Gemfile:
    rails (~> 4.2.0) ruby depends on
      actionview (= 4.2.0) ruby depends on
        activesupport (= 4.2.0) ruby depends on
          i18n (~> 0.7) ruby

    localeapp (~> 1.0.0) ruby depends on
      i18n (< 0.7) ruby
```

**Comment by fabn** [comment]
As a further suggestion you should really use something like [appraisal](https://github.com/thoughtbot/appraisal) to test against different rails versions instead of relying on manually written Gemfiles.

**Comment by michaelbaudino** [comment]
:scream: 
Thanks for mentioning it, it's fixed in `1.0.1` :+1:
